config/registry/schemas/common-types/cron.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$id": "https://app.runx.dev/common-types/cron",
+  "type": "object",
+  "properties": {
+    "commands": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "The command array to execute",
+      "pattern": "^[a-zA-Z0-9-]+$"
+    },
+    "schedule": {
+      "type": "string",
+      "description": "The crontab schedule for this job",
+      "pattern": "^[a-zA-Z0-9 /*-]+$"
+    },
+    "env_vars": {
```

Comment:
I'm questioning if we need this?
why not reuse the same env_vars that is set on the main container?
would that make it so the user doesn't need to update the env vars at 2 places?
or do you have a good use case where we need 2 different set of env vars?

Reply:
I was thinking sometimes a user would like to specify that this is a cron job, but I'm happy to remove it if you think it's a bad idea

Reply:
my preference would be to remove it (and from doc) to keep it as simple as possible. if a user comes with this requirement it's easy to add 👍

Reply:
did you mean to remove it?